### DISCUSSION TITLE
Why is our EITC aggregate a third below other sources?

### DISCUSSION THREAD

**Discussion by MaxGhenis** [PolicyEngine/policyengine-us#1859] [General]
We currently estimate that the [EITC costs $43 billion in 2023](https://policyengine.org/us/policy?focus=policyOutput.netIncome&region=us&timePeriod=2023&baseline=2&reform=5116).

For tax year 2023, Tax-Brain estimates $63 billion in both the [PUF](https://compute.studio/PSLmodels/Tax-Brain/54384/) and [CPS](https://compute.studio/PSLmodels/Tax-Brain/54385/).

[IRS reported](https://www.eitc.irs.gov/eitc-central/statistics-for-tax-returns-with-eitc/statistics-for-tax-returns-with-the-earned-income) $60 billion and $64 billion in EITC for tax years 2020 and 2021, respectively (ARPA expanded EITC for 2021).

Comparing to the raw ASEC's EITC total could help some. Our integration tests match taxsim, so we might be missing some signal from the ASEC.

**Comment by chisquared** [comment]
This shortfall in EITC is a very common result for CPS based microsimulation models -- likely related to differences in household structure as reported in the survey versus tax data. For example, there are usually many fewer head of household units simulated in the CPS than appear as tax returns (because it is beneficial for dual income households to split income by filing as HoH instead of married). Similarly, kids can be reallocated within a household to increase EITC relative to the tax units where a CPS based model places the kids.
